go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/turbaszek/tnijto
+
+go 1.13
```

Comment:
I think we can use 1.14 and perhaps we could go for 1.15 whichc was released august 2020 :)

Reply:
We can, I just need to install it 😃